### PR TITLE
operator nvidia-network-operator (1.1.0)

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

File: operators/nvidia-network-operator/1.1.0/manifests/network-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.7.1+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    description: Deploy and manage NVIDIA networking resources in Kubernetes
+    provider: NVIDIA
+    repository: https://github.com/Mellanox/network-operator/
+    support: NVIDIA
+  name: network-operator.v1.1.0
+  namespace: placeholder
+spec:
```
Comment (fabiendupont) [inline operators/nvidia-network-operator/1.1.0/manifests/network-operator.clusterserviceversion.yaml:47]: The spec does have a `relatedImages` section. Beyond allowing to mirror the images in disconnected environments [1], it allows the certification tooling to identify images for CVE scanning and to perform an in-depth assessment.

That section should not only list the operator image, but also the images the operator will pull to reconcile the CRs, such as the Driver image, the DevicePlugin image, etc...

The images should be identified by their sha256 hash, not by a version tag.

[1] [Enabling your Operator for restricted network environments](https://docs.openshift.com/container-platform/4.9/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs)

File: operators/nvidia-network-operator/1.1.0/manifests/network-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: HostDeviceNetwork
+      name: hostdevicenetworks.mellanox.com
+      version: v1alpha1
+    - kind: MacvlanNetwork
+      name: macvlannetworks.mellanox.com
+      version: v1alpha1
+    - kind: NetworkAttachmentDefinition
```
Comment (fabiendupont) [inline operators/nvidia-network-operator/1.1.0/manifests/network-operator.clusterserviceversion.yaml:57]: Again, I don't think you own this CRD, even though the operator may watch the CRs.

**Comment by e0ne** [comment]
@tonytcampbell, could you please take a look on CI failure? any advice is welcome

**Comment by e0ne** [comment]
u

**Review by fabiendupont — APPROVED** [review]
/lgtm

**Pull request merged**